File: 2308-DivideArrayIntoEqualPairs/2308-DivideArrayIntoEqualPairs.py
# Last updated: 3/24/2026, 11:22:49 AM
class Solution(object):
    def divideArray(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        l=set(nums)
        for i in l:
            if nums.count(i)%2!=0:
                return False
        return True

File: 2308-DivideArrayIntoEqualPairs/2308-DivideArrayIntoEqualPairs.java
// Last updated: 3/24/2026, 11:22:45 AM
class Solution {
    public boolean divideArray(int[] nums) {
        Arrays.sort(nums);
        for(int i=0;i<nums.length;i+=2){
            if(nums[i]!=nums[i+1]){
                return false;
            }
        }
        return true;
    }
}